### ISSUE TITLE
Exclude Folder Option

### ISSUE THREAD

**Issue by DannyCooper** [WPTRT/theme-sniffer#145]
In the case where a theme is using a library (Kirki, TGMPA, etc.) it would be useful to have a text field to enter directories that shouldn't be scanned.

Examples:

```
/vendor
/TGMPA
/inc/kirki
```

**Comment by joyously** [comment]
But libraries must meet the requirements also.

**Comment by DannyCooper** [comment]
Yes, but for universally accepted libraries like the ones I mentioned, they don’t need to be scanned for each theme.

_Assuming they are up to date._

**Comment by joyously** [comment]
I'd rather have them scanned than give the author an option to skip stuff.
And how do you assume they are up to date without scanning?

**Comment by DannyCooper** [comment]
The tool is for reviewers as much as authors. As a reviewer I’m checking the library versioning manually anyway.

**Comment by dingo-d** [comment]
Currently, `vendor`s and `node_modules` are excluded. The reasoning is that you may have those folders during development, but they shouldn't (vendors maybe should in some cases) be scanned as they would most certainly crash the sniffer or report on tons and tons of code that may not end up in the theme in the end.

As for `kirki` and `TGMPA` we should check them, because you never know what kind of code may be included. Somebody can say: I included TGMPA but puts his/hers own evil code inside...
